app-server/app/src/main/java/com/codesoom/myseat/dto/UserResponse.java

```diff
@@ -0,0 +1,22 @@
+package com.codesoom.myseat.dto;
+
+import com.codesoom.myseat.domain.User;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+/**
+ * 예약 정보에 포함되는 사용자 정보
+ */
```

Comment:
```suggestion
/** 예약 정보에 포함되는 사용자 정보 */
```
저희가 정한 코딩 컨벤션에 맞춰서 작성해주시면 좋을 것 같습니다!
https://www.notion.so/e88a5fd4f0b147a5a9ae4f4ca6b88d77